server/pulp/server/controllers/content.py

```diff
@@ -96,3 +126,115 @@ def refresh_content_source(content_source_id=None):
     conduit = ContentSourcesConduit('Refresh Content Source')
     step = ContentSourcesRefreshStep(conduit, content_source_id=content_source_id)
     step.process_lifecycle()
+
+
+@celery.task(base=Task)
```

Comment:
Everything below this is going away in a subsequent PR. These tasks remain from our old design.